content/docs/operating/configuration.md

```diff
@@ -6,66 +6,178 @@ nav_icon: sliders
 # Configuration
 
 Prometheus is configured via command-line flags and a configuration file. While
-the command-line flags configure general system parameters (such as storage
+the command-line flags configure immutable system parameters (such as storage
 locations, amount of data to keep on disk and in memory, etc.), the
 configuration file defines everything related to scraping [jobs and their
 instances](/docs/concepts/jobs_instances/), as well as which [rule files to
 load](/docs/querying/rules/#configuring-rules).
 
-To view all available command-line flags, run Prometheus with the `-h` option.
+To view all available command-line flags, run `prometheus -h`.
+
+
+## Configuration file
 
 To specify which configuration file to load, use the `-config.file` flag.
-A configuration file is written in ASCII protocol buffer form, and every
-available option is explained in detail in the protocol buffer schema
-definition at
-https://github.com/prometheus/prometheus/blob/master/config/config.proto.
+The configuration file (including rule files) can be reloaded at runtime by 
+sending SIGHUP to the Prometheus process.
+
+The file is written in [YAML format](http://en.wikipedia.org/wiki/YAML),
+defined by the scheme described below.
+Brackets indicate that a parameter is optional. For non-list parameters the
+value is set to the specified default.
+
+Generic placeholders have the following requirements:
+
+`<duration>`: fulfill the regular expression `[0-9]+[smhdwy]`
```

Comment:
How about "a duration matching the regular expression ..." and similar for the others, like "a string of unicode characters" and "a valid path in the current directory", without the awkward verb in the beginning.